.github/workflows/pull-requests.yml

```diff
@@ -0,0 +1,22 @@
+name: Pull Requests
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
```

Comment:
Not sure if we really need multiple node versions for this project but open to your thoughts here.